src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/WikiBasePageObject.java

```diff
@@ -167,15 +158,10 @@ public HistoryPagePageObject openFileHistoryPage(String articlePage, String wiki
     return new HistoryPagePageObject(driver);
   }
 
-  public RegisterPage openSpecialUserSignUpPage(String wikiURL) {
-    getUrl(wikiURL + URLsContent.SPECIAL_USER_SIGNUP);
+  public AttachedRegisterPage openSpecialUserSignUpPage(String wikiURL) {
```

Comment:
this method just navigates to `/register`, which is only used in context of `AttachedRegisterPage`, so I'll leave it as it is